about/our-team.php

```diff
@@ -189,12 +208,12 @@
                 <!-- Social Media Manager -->
                 <div class="cell-lg-12 bio-area">
                     <div class="range range-sm-middle">
-                        <div class="cell-md-3"><img src="/images/bio-pics/ally-flores.jpg" alt="" width="360" height="404" class="img-responsive" />
+                        <div class="cell-md-3"><img src="" alt="" width="360" height="404" class="img-responsive" />
```

Comment:
Use the temp pic in bio-pics folder under images for an image

Description for this
"I'm a freshman at UNT majoring in mechanical and engineering. I was in my high school robotics team and am excited to continue doing robotics throughout college. In my free time, I like reading, drawing, or walking my dog!"